presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -150,19 +157,6 @@ public PinotConfig setControllerUrls(String controllerUrl)
         return this;
     }
 
-    @Nullable
-    public String getRestProxyUrl()
-    {
-        return restProxyUrl;
-    }
-
-    @Config("pinot.rest-proxy-url")
-    public PinotConfig setRestProxyUrl(String restProxyUrl)
```

Comment:
@nausicaasnow Can you help review whether this property is used by your side? We are trying to reduce the config values to avoid confusion, and the `pinot.rest-proxy-url` value is merged to be the controllerURL (hide the rest-proxy concept)

Reply:
cc @dharakk @agrawaldevesh This requires pinot proxy to implement the same API for controller(fetching table configs/schemas/etc) and broker to fetch routingTable, etc.